website/www/site/layouts/partials/hooks/body-end.html

```diff
@@ -0,0 +1,17 @@
+{{/*
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+*/}}
+
+<script src="/js/hero/lottie-light.min.js"></script>
+<script src="/js/hero/hero-desktop.js"></script>
+<script src="/js/hero/hero-mobile.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/keen-slider@5.3.2/keen-slider.min.js"></script>
+<script src="/js/quotes-slider.js"></script>
```

Comment:
Yeah. As you said one of the reason was not to delay the loading of visible content. In this case, there is another reason. For example, `quotes-slider.js` script needs to wrap an element with the id `my-keen-slide` and work on it, then render a new element with such sliding feature. Thus, with this case, the script should be put at the end to wait for the element `my-keen-slider` appears first so that it'll be able to get that element.